### PR TITLE
Update External Links To Internal Links In Docs

### PR DESCRIPTION
**Pull request by Matt-Cumber**
**Description of work.**
First part of cleaning up external links. This part replaces external links with internal links where possible. In the original issue I have listed most of the remaining links which need attention (I may have missed some), and these can be addressed in future pr's

**Do not close the original issue or link via zenhub**

**To test:**
Build docs and dev-docs, are the links okay?
Do have a quick look for other possible links that can be easily swapped to internal links, however as this is only part of the overall maintenance the remaining few can be picked up in future pr's

Fixes part of #31340 

*This does not require release notes* because **it is documentation maintenance**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/algorithms/StartLiveData-v1.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ simply calls :ref:`algm-LoadLiveData` at a fixed interval.
    For details on the way to specify the data processing steps, see
    :ref:`LoadLiveData <algm-LoadLiveData>`.
 
-Instructions for setting up a "fake" data stream are found `here
+Instructions for setting up a "fake" data sNtream are found `here
```
Comment (robertapplin) [inline docs/source/algorithms/StartLiveData-v1.rst:23]: Is `sNtream` a typo?

**Review by robertapplin — CHANGES_REQUESTED** [review]
The docs build fine, good job. Just have one potential typo to fix and then I'll approve this

File: docs/source/algorithms/StartLiveData-v1.rst (hunk 1 above)
Comment (Matt-Cumber) [reply docs/source/algorithms/StartLiveData-v1.rst:23]: It looks like a typo to me, I must have accidentally added this. Good spot.

**Review by robertapplin — DISMISSED** [review]
Looks good, building docs and dev-docs works as expected

File: docs/source/release/v5.1.0/mantidplot.rst [hunk 2]
```diff
@@ -23,7 +23,7 @@ This decision was due to the lack of support for Qt4 and the many major problems
 Also, MantidPlot has not received any updates or many bug fixes for the last year, and as such Workbench is a more stable and
 functional user interface in many areas.
 
-We have created a high level `comparison of the functionality <https://www.mantidproject.org/Workbench_Feature_Comparison>`_
+We have created a high level `comparison of the functionality <https://archive.mantidproject.org/Workbench_Feature_Comparison>`_
```
Comment (DavidFair) [inline docs/source/release/v5.1.0/mantidplot.rst:26]: This is still at www.

File: docs/source/release/v3.7.1/framework.rst [hunk 3]
```diff
@@ -59,7 +59,7 @@ New
   parameters (DIFA, DIFC, TZERO) from peaks fitted using
   :ref:`EnggFitPeaks <algm-EnggFitPeaks>`.
 - :ref:`FindEPP <algm-FindEPP>` This algorithm performs Gaussian fit to find the elastic peak position.
-  As a result, `TableWorkspace <http://www.mantidproject.org/TableWorkspace>`_ with parameters of the fitted peaks is created.
+  As a result, `TableWorkspace <https://archive.mantidproject.org/TableWorkspace>`_ with parameters of the fitted peaks is created.
```
Comment (DavidFair) [inline docs/source/release/v3.7.1/framework.rst:62]: This should point to the internal docs TableWorkspace (if we have one)

File: docs/source/release/v3.12.0/direct_inelastic.rst [hunk 4]
```diff
@@ -48,7 +48,7 @@ Crystal Field
 New features
 ############
 
-- The crystal field computation and fitting engine is now feature complete. It can now handle multi-site computation and simultaneous fitting of inelastic spectra and physical properties dataset. See the :ref:`Crystal Field Python Interface` help page for details, and `<http://www.mantidproject.org/Crystal_Field_Examples>`_ for examples of use.
+- The crystal field computation and fitting engine is now feature complete. It can now handle multi-site computation and simultaneous fitting of inelastic spectra and physical properties dataset. See the :ref:`Crystal Field Python Interface` help page for details, and `<https://archive.mantidproject.org/Crystal_Field_Examples>`_ for examples of use.
```
Comment (DavidFair) [inline docs/source/release/v3.12.0/direct_inelastic.rst:51]: Should be `www.` not `archive` ditto on line 36 too

**Review by DavidFair — CHANGES_REQUESTED** [review]
Some links to "unfix" as they are still on the original host

Thanks for going through these, there was a large number and it genuinely makes a difference for users browsing the documentation

**Comment by Matt-Cumber** [comment]
> Some links to "unfix" as they are still on the original host
> 
> Thanks for going through these, there was a large number and it genuinely makes a difference for users browsing the documentation

Hi David, I asked @martyngigg what to do with old release notes and it was decided to replace these links by pointing them to the archive version. If we can just check if this is still the plan before I change them back that would be great

**Comment by DavidFair** [comment]
Ah, the plan is to move stuff from the wiki at `www` to `archive.mantidproject.org` once wok on some new web pages has been completed
But until then were trying to leave them at the original URLs, so I guess the question is do we break them now for the future, or leave them working and fix them later?

I'm leaning towards leave them now and fix later as were unsure when the migration will happen

**Comment by Matt-Cumber** [comment]
What do you mean by break? as the URL's with `archive` at the beginning still work and and take you to the exact same page as the `www` version. Just double checking we're on the same page (no pun intended)

**Comment by DavidFair** [comment]
> What do you mean by break? as the URL's with `archive` at the beginning still work and and take you to the exact same content as the `www` version. Just double checking we're on the same page (no pun intended)

You're completely correct, the worst part is I setup `archive` to point to the same content as `www` and completely forgot :man_facepalming: 

I'll mark those comments as resolved, sorry about the confusion. Regarding the table workspace do we have any user docs instead of wiki docs? That's the only thing left

**Comment by Matt-Cumber** [comment]
Thanks for taking a look at this @DavidFair and @robertapplin, I hope everything is sorted now 👍

**Review by DavidFair — APPROVED** [review]
LGTM, lets :shipit:

**Pull request merged**